src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature

```diff
@@ -92,17 +102,16 @@ Feature: Validation of the structure of the URLs
                       "correlationId": null
                   }
                   """
-              And the response headers includes
+              And the response headers include
                   """
-                  Content-Type: application/problem+json
+                    {
+                        "Content-Type": "application/problem+json"
+                    }
```

Comment:
The old format was invalid, as the step definition expects the headers to be defined in JSON format. We didn't notice the error because the step is being ignored.